src/utils/__tests__/uploads.test.js

```diff
@@ -184,6 +186,82 @@ describe('util/uploads', () => {
         });
     });
 
+    describe('getPackageFileFromDataTransferItem()', () => {
+        test('should return file of UploadFileWithAPIOptions type when itemData is UploadDataTransferItemWithAPIOptions type', async () => {
+            const mockPackageItem = {
+                getAsFile: jest.fn(() => mockFile),
+                webkitGetAsEntry: () => entry,
+            };
+            const itemData = {
+                item: mockPackageItem,
+                options,
+            };
+
+            expect(await getPackageFileFromDataTransferItem(itemData)).toEqual({
+                file: mockFile,
+                options,
+            });
+        });
+    });
+
+    describe('isDataTransferItemAPackage()', () => {
+        test('should be false if data transfer item thinks it is a folder', () => {
+            const folderEntry = {
+                isDirectory: true,
+            };
+
+            const folderItem = {
+                kind: '',
+                webkitGetAsEntry: () => folderEntry,
+            };
+
+            const itemData = {
+                item: folderItem,
+                options,
+            };
+
+            expect(isDataTransferItemAPackage(itemData)).toBeFalsy();
+        });
+
+        test('should be false if data transfer item thinks it is a file', () => {
+            const fileEntry = {
+                isDirectory: false,
+                isFile: true,
+            };
+
+            const folderItem = {
```

Comment:
`fileItem`? Also, could this test be combined with the test above into a `test.each` loop?

Reply:
this one is a bit funny. for package files, it identifies as both a file AND a folder, so I used the confusing name to represent this on purpose. maybe the added comments could be clearer about that in the code.